### PR TITLE
#93 deploy script

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

File: coda-prototype/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Delete last attempt if it exists
```
Comment (lukechurch) [inline coda-prototype/deploy.sh]: As discused lets drop this

File: coda-prototype/initUI.js [hunk 2]
```diff
@@ -394,6 +394,25 @@ function initUI(dataset) {
     var messagePanel = $("#message-panel");
     var editorRow = $("#editor-row");
 
+    $.getJSON("version.json", version => {
+        let hash = version.hash === "develop" ? "develop" : `${version.hash.substring(0, 6).toUpperCase()}`;
```
Comment (lukechurch) [inline coda-prototype/initUI.js]: As discussed let's refactor this to unify ln398 and ln400

File: coda-prototype/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (as2388) [reply coda-prototype/deploy.sh]: done

File: coda-prototype/initUI.js (hunk 2 above)
Comment (as2388) [reply coda-prototype/initUI.js]: done

**Comment by as2388** [comment]
Will merge after #95 is reviewed and merged.

**Pull request merged**